mockall/src/lib.rs

```diff
@@ -1368,7 +1368,7 @@ pub use mockall_derive::concretize;
 /// }
 /// # fn main() {}
 /// ```
-/// Mocking an unsupported `#[derive(X)]` attribute, e.g. `Clone`, is
+/// Mocking an unsupported `#[derive(X)]` attribute, e.g. [`Clone`], is
```

Comment:
Does this link actually work?  Did you generate the docs to check?  And is it even worthwhile to make a link here? I don't think many readers will need one.

Reply:
Yes, the links are valid. I've added them for the sake of consistency